### DISCUSSION TITLE
Hidden GitHub in WhatsApp account.

### DISCUSSION THREAD

**Discussion by Wellesle** [community/community#134664] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### GitHub Feature Area

Security and Privacy

### Body

Why would my husband have a GitHub account hidden in his WhatsApp account?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Wellesle ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by FlutterSmith** [comment]
It might be an accidental or unnecessary link that they forgot about.
